packages/core/upload/server/src/controllers/validation/admin/folder-file.ts

```diff
@@ -53,40 +54,39 @@ const validateDuplicatesMoveManyFoldersFilesSchema = yup
     return true;
   });
 
-const validateMoveFoldersNotInsideThemselvesSchema = yup
-  .object()
-  .test(
-    'dont-move-inside-self',
-    'folders cannot be moved inside themselves or one of its children',
-    async function (value) {
-      const { folderIds, destinationFolderId } = value;
-      if (destinationFolderId === null || isEmpty(folderIds)) return true;
-
-      const destinationFolder = await strapi.db.query(FOLDER_MODEL_UID).findOne({
-        select: ['path'],
-        where: { id: destinationFolderId },
-      });
+const validateMoveFoldersNotInsideThemselvesSchema = yup.object().test(
+  'dont-move-inside-self',
+  'folders cannot be moved inside themselves or one of its children',
+  // eslint-disable-next-line func-names
+  async function (value) {
```

Comment:
same thing about arrow function

Reply:
I preferred to solve the problem by giving a name to the function